python/perspective/perspective/core/widget.py

```diff
@@ -52,10 +53,10 @@ class PerspectiveWidget(Widget, PerspectiveViewer):
     # Required by ipywidgets for proper registration of the backend
     _model_name = Unicode('PerspectiveModel').tag(sync=True)
     _model_module = Unicode('@finos/perspective-jupyterlab').tag(sync=True)
-    _model_module_version = Unicode('^0.3.9').tag(sync=True)
+    _model_module_version = Unicode('^0.4.0-rc.1').tag(sync=True)
```

Comment:
We need to brainstorm something for this down the road, and try to avoid breaking changes in patch releases

Reply:
maybe the `version_python` script would work here